msnc/model.py

```diff
@@ -23,7 +23,6 @@ def __init__(
         checkpoint_interval=10,
         batch_size=32,
         seed=1,
-        save_best_model=True,
```

Comment:
I prefer that it does not remove immediately but warn users to navigate specifying `save_best_model` in `run_training`. (But of course, it is OK for me to current patch.)

Reply:
Thank you! So, I merge this PR soon because the number of users are still small 😅